compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/SkiaImageAsset.skiko.kt

```diff
@@ -71,6 +71,12 @@ fun ImageBitmap.asSkiaBitmap(): Bitmap =
         else -> throw UnsupportedOperationException("Unable to obtain org.jetbrains.skia.Image")
     }
 
+fun ImageBitmap.asByteArray(): ByteArray =
```

Comment:
Do we need it as a public API? If so, let's make it separately (+ reviewer, + docs)

Reply:
@MatkovIvan that's a valid question that bothers me as well. We need it in exactly one place but well, we need it. 
The alternatives are following: 

- To have a separate private implementation in `WebDragAndDropManager` but there will be code duplication
- To rely on method `SkiaBackedImageBitmap::readPixels` and then convert back from IntArray to ByteArray

I'm not  extremely excited about both options but second one feels a slightly better alternative, so @eymar and @MatkovIvan what do you think?

Reply:
One more option is move this function to web source set, keeping it public by mark it with `InternalComposeUiApi`. 

WDYT?

We use that annotation for some things, like `ComposeSceneContext`

Reply:
@eymar this is close to keeping separate private implementation because the whole point was to reuse `SkiaBackedImageBitmap::readPixelsAsByteArray` in `SkiaBackedImageBitmap::readPixels`  - but honestly, my friends, let's just use the IntArray version - the prices of conversion is negligibly low and even more so, after some changes in skiko it might be possible that we won't need to have this conversion at all.

Reply:
Alright.

Just want to say that:
> to reuse SkiaBackedImageBitmap::readPixelsAsByteArray in SkiaBackedImageBitmap::readPixels

they can be reused if made `internal` instead of `private`

Reply:
@eymar no, it's in a different module